packages/gasket-plugin-workbox/package.json

```diff
@@ -13,9 +13,7 @@
     "test": "cross-env NODE_OPTIONS='--unhandled-rejections=strict' jest",
     "test:watch": "jest --watchAll",
     "test:coverage": "jest --coverage",
-    "posttest": "npm run lint",
-    "typecheck:skip": "tsc",
-    "typecheck:watch": "tsc --watch"
```

Comment:
Do you want to leave skips these so we remember to address in v7? And/or do you otherwise have a ticket to track?

Reply:
Currently, I have an array of all of the skipped packages in the align-packages.js file. Will definitely spin up a ticket to address these in v7.